odoo/service/server.py

```diff
@@ -1348,12 +1348,12 @@ def preload_registries(dbnames):
     for dbname in dbnames:
         try:
             threading.current_thread().dbname = dbname
-            update_module = config['init'] or config['update']
+            update_module = do_update = config['init'] or config['update']
```

Comment:
Thanks for the fix, it is really an issue.
one nitpicking
`update_module` -> [`update_from_config`](https://github.com/odoo/odoo/blob/master/odoo/modules/loading.py#L266)
`do_update` -> `update_module`

Reply:
Renaming `update_module` would imply changes in other lines (that were last edited years ago), are we sure we want that? It will hide older history for little gain IMO.